aperturedb/BBoxLoader.py

```diff
@@ -1,63 +1,69 @@
 from aperturedb import ParallelLoader
 from aperturedb import CSVParser
 
-HEADER_X_POS  = "x_pos"
-HEADER_Y_POS  = "y_pos"
-HEADER_WIDTH  = "width"
+HEADER_X_POS = "x_pos"
```

Comment:
what happened here? it used to look so nice all with equal indentation, easier to read. Maybe so automatic rule from your editor?

Reply:
Not an editor, I used this https://black.readthedocs.io/en/stable/index.html . This seems like quite a mature and commonly used tool in OSS python projects. It has plugins for tons of editors, but it also integrates with github actions and commit prehook.

It is opinionated ...  yes, but so is python and pydoc standards.

So what this would offer is hassle from manually conforming to a consistent coding style.
We might not see it's immediate benefit as the number of people working with the repo is less, but you can immediately see there is one less thing to worry about as in the comment from docs/conf.py . I had not run it on root, but at aperturedb level. 

This makes even more sense as the repo is public, and it's hard to align to a common standard.
This tool just automates the process, without consuming any mental energy.

Reply:
I love the idea of start using a style checker/enforcer for our Python projects. 

It would be great to enforce PEP8. Black is PEP8 compliant. But I think Black would force many style changes on top of PEP8 that are arbitrary rather than functional. Another thing is that Black is not very configurable, so it would mean using their style or die.

PEP8 is less restrictive and oriented towards readability. That would make it a great code style to enforce.
Here is a package that does that: https://pypi.org/project/pep8/
We can add a check on our CI and enforce it.

Here is another one, recommended by the very same Guido van Rossum: https://github.com/PyCQA/flake8

Reply:
Flake8 does look like a promising option. Let me try to change apply auto formatting using that.